src/components/Navigation/Header/TopBalance/BalanceBox/BalanceBox.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Box, Typography } from "@material-ui/core";
+
+/**
+ * @typedef {Object} DefaultProps
+ * @property {JSX.Element|String} name
+ * @property {JSX.Element|String} [value1]
+ * @property {JSX.Element|String} [value2]
+ */
+
+/**
+ * Render a part of the equity bar
+ *
+ * @param {DefaultProps} props Component props.
+ * @returns {JSX.Element} JSX
+ */
+const BalaneBox = ({ name, value1, value2 }) => {
```

Comment:
typo but not important..

Reply:
Also it's hard to guess what is value1 and value2, in the future let's try to give the component param distinguishable names (or at least document what they represent)

Reply:
yeah, I will refactor these in the future and document them for now.